corehq/warehouse/transforms/__init__.py

```diff
@@ -0,0 +1,17 @@
+from corehq.warehouse.const import (
+    APP_STATUS_FACT_SLUG,
+)
+
+
+def app_status_transform(row):
+    print row
```

Comment:
so i definitely assumed we'd have a portion where we'd be able to run some python function over a row in order to perform transformations. some of the transforms get quite tricky, for example of getting the commcare version, you can see this function: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/receiverwrapper/util.py#L146. maybe this is me just misunderstanding the way these things work, but couldn't we still do bulk sql statements if we iterate over the result set and then do another bulk update?